F3FChrono/chrono/Sound.py

```diff
@@ -79,11 +85,13 @@ def sound_time(self, run_time):
 
     def sound_time(self, run_time):
         if (self.play_voice):
-            self.voice_engine.setProperty('voice', "english")
-            #self.voice_engine.setProperty('voice', "french")
             self.voice_engine.say("{:0>.2f}".format(run_time))
             self.start()
 
+    def pilot(self, first_name, name):
+        if self.play_voice:
+            self.voice_engine.say("Wait competitor " + first_name + " " + name)
```

Comment:
Maybe we could just give the pilots name  ?
Or say only  "Next pilot :"   + first_name + " " + name ?

Reply:
done with "Next pilot :" + first_name + " " + name